vnext/Microsoft.ReactNative/Fabric/Composition/CompositionDynamicAutomationProvider.cpp

```diff
@@ -135,7 +135,7 @@ HRESULT __stdcall CompositionDynamicAutomationProvider::GetPatternProvider(PATTE
 long GetControlType(const std::string &role) noexcept {
   if (role == "adjustable") {
     return UIA_SliderControlTypeId;
-  } else if (role == "group" || role == "search" || role == "radiogroup" || role == "timer" || role.empty()) {
```

Comment:
The [aria core aam](https://www.w3.org/TR/core-aam-1.1/#mapping_role_table) says that search should map to group and searchbox should map to edit, but search is also unlisted in the official react-native role prop value enum: https://reactnative.dev/docs/accessibility#role

I recommend leaving as is or removing entirely.